test/e2e/testsuite/testsuite.go

```diff
@@ -1074,6 +1076,32 @@ func EnsureKubeConfigIsCleanedUp() error {
 	return nil
 }
 
+func EnsureCRCManPagesAvailability(expectedStatus string) error {
+	expectedManFileList := []string{
+		"crc-bundle-generate.1.gz", "crc-config.1.gz", "crc-start.1.gz",
+		"crc-bundle.1.gz", "crc-console.1.gz", "crc-status.1.gz",
+		"crc-cleanup.1.gz", "crc-delete.1.gz", "crc-stop.1.gz",
+		"crc-config-get.1.gz", "crc-ip.1.gz", "crc-version.1.gz",
+		"crc-config-set.1.gz", "crc-oc-env.1.gz", "crc.1.gz",
+		"crc-config-unset.1.gz", "crc-podman-env.1.gz",
+		"crc-config-view.1.gz", "crc-setup.1.gz",
+	}
+	userHomeDir, err := os.UserHomeDir()
+	if err != nil {
+		return fmt.Errorf("failed to get user home directory: %v", err)
+	}
+	manPageDir := filepath.Join(userHomeDir, ".local", "share", "man", "man1")
+	for _, manPage := range expectedManFileList {
+		manFile := filepath.Join(manPageDir, manPage)
+		manCommand := fmt.Sprintf("man -P cat %s", manFile)
```

Comment:
should we be checking `man -P cat crc` instead of using the manpage filepath? 

i think the generation of the files is already covered by the unit tests and here we can check for if `crc setup` also updated the required env variables for having the manpages directly accessible by `man`

Reply:
I added this instead of directly checking the command name because `man` wasn't discovering the `~/.local/share/man/man1` directory on the QE lab instance. There was also a problem with MacOS as there was a package named `crc` that had a manpage (see https://github.com/crc-org/crc/issues/4608#issuecomment-2650598324)

Reply:
additionally providing the manpage section number seems to access the correct `crc` manpage, 
this works: `man -P cat 1 crc`

Reply:
okay, I've updated it as per your suggestion.